encodedcc.py

```diff
@@ -418,7 +434,8 @@ def setup(self):
                 self.header.sort()
             elif self.args.field:
                 if os.path.isfile(self.args.field):
-                    self.fields = [line.strip() for line in open(self.args.field)]
```

Comment:
I actually prefer this all in 1 line but I am not going to be a stickler about it.

Reply:
Possibly. This is the automatic PEP8 linting that happens every time I hit save.